### ISSUE TITLE
value range of loss_mse, loss_ssim, loss_content in training

### ISSUE THREAD

**Issue by Jillian2017** [aiff22/PyNET#6]
Hi:
Thanks a lot for sharing your source code. 
In your paper, the loss for level 1 is loss_1 = L_vgg + 0.75*L_ssim + 0.05*L_mse, and each loss is normalized to 1. While in your code, I think the condition of  LEVEL ==0 is same as loss_1 in your paper. The loss is loss_generator = loss_mse*20 + loss_content + (1 - loss_ssim)*20.
Q1, why are they different? How to normalize L_ms to 1? 
Q2, why is these weights for loss_mse and (1 - loss_ssim) the same. I think loss_mse is a big value(>50), while 1-ssim is smaller than 0.1.  
Q3, could you tell me the value range of these 3 losses? Maybe this can help me to understand the weights for loss. In my case, the loss_mse is around 100, ssim is around 0.98, and loss_content is around 7.
Thanks in advance.

**Comment by aiff22** [comment]
@Jillian2017,

You can find the information regarding your question in the [following issue](https://github.com/aiff22/PyNET-PyTorch/issues/1). If you check the codes used for training the model, you can see that the MSE loss is [normalized per pixel](https://github.com/aiff22/PyNET/blob/9333ac59b5270445725f75317dbb0a3fc3aee688/train_model.py#L63), therefore its value is significantly smaller than 1. Please note that all losses [are printed during the training](https://github.com/aiff22/PyNET/blob/9333ac59b5270445725f75317dbb0a3fc3aee688/train_model.py#L185), therefore one can monitor their ranges when running the model.
